### ISSUE TITLE
Conceptual Questions around Concept Vector and Dot Product

### ISSUE THREAD

**Issue by drishyamlabs** [pnxenopoulos/cav-keras#5]
Hi Peter,

Thanks a lot for sharing the valuable codes. I have few basic questions:
1) As per the research paper, concept vector is orthogonal to the decision boundary. Can you please guide us where in the code is that happening?
2) In the original implementation (https://github.com/tensorflow/tcav/blob/master/tcav/tcav.py) line 86, tcav score is defined as "TCAV score (i.e., ratio of pictures that returns negative dot product wrt loss).". However in this implementation, we are taking positive dot product. Can you please help in the clarification/ differences in the implementation. I am really hard time spot the differences.
3) Do you also have any implementation around DTCAV?
